### PR TITLE
KOGITO-6471 DMN [JIT Executor] Add file `path` to validation messages

### PR DESCRIPTION
**Pull request by tarilabs**
**JIRA**: https://issues.redhat.com/browse/KOGITO-6471

**referenced Pull Requests**: 

* https://github.com/kiegroup/drools/pull/4153
* https://github.com/kiegroup/kogito-apps/pull/1210

<details>
<summary>
How to replicate CI configuration locally?
</summary>

Build Chain tool does "simple" maven build(s), the builds are just Maven commands, but because the repositories relates and depends on each other and any change in API or class method could affect several of those repositories there is a need to use [build-chain tool](https://github.com/kiegroup/github-action-build-chain) to handle cross repository builds and be sure that we always use latest version of the code for each repository.
 
[build-chain tool](https://github.com/kiegroup/github-action-build-chain) is a build tool which can be used on command line locally or in Github Actions workflow(s), in case you need to change multiple repositories and send multiple dependent pull requests related with a change you can easily reproduce the same build by executing it on Github hosted environment or locally in your development environment. See [local execution](https://github.com/kiegroup/github-action-build-chain#local-execution) details to get more information about it.
</details>

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>Pull Request</b>  
  Please add comment: <b>Jenkins retest this</b>
 
* <b>Quarkus LTS checks</b>  
  Please add comment: <b>Jenkins run LTS</b>

* <b>Native checks</b>  
  Please add comment: <b>Jenkins run native</b>

</details>

### CODE REVIEW

**Comment by tarilabs** [comment]
Jenkins retest this

File: jitexecutor/jitexecutor-dmn/src/test/java/org/kie/kogito/jitexecutor/dmn/api/MultipleModelsTest.java [hunk 1]
```diff
@@ -55,6 +70,21 @@
     private static ResourceWithURI ch11model1;
     private static ResourceWithURI ch11model2;
 
+    private static final ObjectMapper MAPPER;
```
Comment (danielezonca) [inline jitexecutor/jitexecutor-dmn/src/test/java/org/kie/kogito/jitexecutor/dmn/api/MultipleModelsTest.java:73]: Have you tried to inject ObjectMapper and use the Quarkus one? I expect it works and this should help keep this ObjectMapper in sync with future changes

File: jitexecutor/jitexecutor-dmn/src/main/java/org/kie/kogito/jitexecutor/dmn/responses/JITDMNResult.java [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+import org.kie.dmn.api.core.DMNContext;
+import org.kie.dmn.api.core.DMNDecisionResult;
+import org.kie.dmn.api.core.DMNMessage;
+import org.kie.dmn.api.core.DMNMessage.Severity;
+import org.kie.dmn.core.internal.utils.MapBackedDMNContext;
+import org.kie.dmn.core.internal.utils.MarshallingStubUtils;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+public class JITDMNResult implements Serializable,
```
Comment (danielezonca) [inline jitexecutor/jitexecutor-dmn/src/main/java/org/kie/kogito/jitexecutor/dmn/responses/JITDMNResult.java:36]: Can you please clarify why have you decided to re-implement `DMNResult` (and the other interfaces)?

**Review by danielezonca — APPROVED** [review]

File: jitexecutor/jitexecutor-dmn/src/test/java/org/kie/kogito/jitexecutor/dmn/api/MultipleModelsTest.java (hunk 1 above)
Comment (tarilabs) [reply jitexecutor/jitexecutor-dmn/src/test/java/org/kie/kogito/jitexecutor/dmn/api/MultipleModelsTest.java:73]: This is about the tests Daniele, these are the code changes which are not required by Quarkus, these are the mapper config which a java-client should configure when interacting with this REST api.
Eg: this is something the Kie Server Java Client does internally.

File: jitexecutor/jitexecutor-dmn/src/main/java/org/kie/kogito/jitexecutor/dmn/responses/JITDMNResult.java (hunk 2 above)
Comment (tarilabs) [reply jitexecutor/jitexecutor-dmn/src/main/java/org/kie/kogito/jitexecutor/dmn/responses/JITDMNResult.java:36]: Conflicting requirements: DMNResult is the interface with the common methods, which is not anyway processed by Jackson.

On Kogito, we don't display the path.
On JIT Runner, we have now received the requirement to display the path.

**Pull request merged**